packages/mobile/src/screens/wallet-screen/components/CoinCard.tsx

```diff
@@ -5,25 +5,38 @@ import { useFeatureFlag, useFormattedTokenBalance } from '@audius/common/hooks'
 import { FeatureFlags } from '@audius/common/services'
 import { Image, TouchableOpacity } from 'react-native'
 
-import { Flex, HexagonalIcon, Skeleton, Text } from '@audius/harmony-native'
+import {
+  Box,
+  Flex,
+  HexagonalIcon,
+  Skeleton,
+  Text
+} from '@audius/harmony-native'
 import { useNavigation } from 'app/hooks/useNavigation'
 
-// TODO: Fix loading states
-// Skeletons are not working as expected
-
 const ICON_SIZE = 64
 
 const CoinCardSkeleton = () => {
   return (
     <Flex column gap='xs'>
-      <Skeleton w='240px' h='36px' />
-      <Skeleton w='140px' h='24px' />
+      <Box w={240} h={36}>
+        <Skeleton />
+      </Box>
+      <Box w={140} h={24}>
+        <Skeleton />
+      </Box>
```

Comment:
yea I noticed that this was necessary in another area. Annoying that skeletons need to be wrapped like this to work. We should update the component